node-client/src/config.ts

```diff
@@ -273,12 +273,10 @@ export class KubeConfig {
                 const expiration = Date.parse(expiry);
                 if (expiration < Date.now()) {
                     if (config['cmd-path']) {
-                        let cmd = '"' + config['cmd-path'] + '"';
-                        if (config['cmd-args']) {
-                            cmd = cmd + ' ' + config['cmd-args'];
-                        }
+                        const args = config['cmd-args'] ? [config['cmd-args']] : [];
                         // TODO: Cache to file?
-                        const result = shelljs.exec(cmd, { silent: true });
+                        // TODO: do this asynchronously
+                        const result = execa.sync(config['cmd-path'], args);
```

Comment:
I don't know the inner workings of `shelljs.exec` but the name implies it used to go via the shell.  Should we therefore use `execa.shellSync`?  (This is a question, not a suggestion... I lack knowledge and would value your advice.)